go.sum

```diff
@@ -40,16 +40,22 @@ github.com/0chain/common v0.0.6-0.20230127095721-8df4d1d72565 h1:z+DtCR8mBsjPnEs
 github.com/0chain/common v0.0.6-0.20230127095721-8df4d1d72565/go.mod h1:UyDC8Qyl5z9lGkCnf9RHJPMektnFX8XtCJZHXCCVj8E=
 github.com/0chain/errors v1.0.3 h1:QQZPFxTfnMcRdt32DXbzRQIfGWmBsKoEdszKQDb0rRM=
 github.com/0chain/errors v1.0.3/go.mod h1:xymD6nVgrbgttWwkpSCfLLEJbFO6iHGQwk/yeSuYkIc=
-github.com/0chain/gosdk v1.10.1-0.20231009155148-2f6b57548450 h1:v997C2HUfq63KRtid8T0yUPssVnmxsAI8Oah2dVivJo=
-github.com/0chain/gosdk v1.10.1-0.20231009155148-2f6b57548450/go.mod h1:6Co9xV9/jeS9Bdi0L3fISOnKbrm0qPTzRtBhIdB/yqY=
+github.com/0chain/gosdk v1.8.18-0.20230902144943-ec223996673c h1:EkFJlu3vui3myAjvWyXcgpqgbU5vrRdamuTgKWPd1e4=
+github.com/0chain/gosdk v1.8.18-0.20230902144943-ec223996673c/go.mod h1:/AYFyM6A7rBWk/AoAy/oClmNnpdznvA9d7ND2rhSGJI=
```

Comment:
Please update gosdk to merge the latest changes from sprint-1.11, otherwise latest changes in gosdk will be lost

Reply:
Done